features/step_definitions/search_functionality_steps.rb

```diff
@@ -0,0 +1,31 @@
+Given(/^that i am on the landing page$/) do
```

Comment:
You can ignore this comment. I will update the configs to not have this in future